### DISCUSSION TITLE
Udelade optionen ”Check automatisk for opdateringer” under installationen

### DISCUSSION THREAD

**Discussion by vucdjurs** [Eduap-com/WordMat#106] [Support]
Jeg vil forsøge at udrulle Wordmat med Intune(MS Endpoint Manager) og i den forbindelse vil jeg gerne udelade optionen ”Check automatisk for opdateringer” under installationen men ved ikke hvordan det kan gøres. Hvordan kan jeg tilrette installations kommandoen "WordMat125-2.exe /silent ..."  så der ikke tjekkes for opdateringer? Det kunne også være godt at medtage at den tidligere installation afinstalleres så brugerne ikke skal tage stilling til noget som helst.

**Comment by Eduap-com** [comment]
Jeg kender ikke til intune. Den næste version af WordMat vil ikke slå automatiske opdateringer til som standard.

Hvis du kan redigere registreringsdatabasen kan du sætte følgende til 0
HKEY_CURRENT_USER\SOFTWARE\WordMat\Settings\CheckForUpdate
